admin_run.go

```diff
@@ -114,7 +112,10 @@ func (s *adminRuns) ForceCancel(ctx context.Context, runID string, options Admin
 }
 
 // Check that the field RunStatus has a valid string value
-func (o AdminRunsListOptions) valid() error {
+func (o *AdminRunsListOptions) valid() error {
+	if o == nil { //no need to validate fields
```

Comment:
```suggestion
	if o == nil { // no need to validate fields
```

would our new linters do it like this?

Reply:
Yes, not sure which lint library, but one of them forces you to add a space in between the `//` and the comment